src/components/Toast/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { ReactNode } from 'react';
+
+import {
+  Box,
+  ButtonGroup,
+  Card,
+  CardBody,
+  Flex,
+  Heading,
+  IconButton,
+} from '@chakra-ui/react';
+import i18n from 'i18next';
+import { LuCheckCircle2, LuInfo, LuX, LuXCircle } from 'react-icons/lu';
+import { ExternalToast, toast } from 'sonner';
+import { match } from 'ts-pattern';
+
+import { Icon } from '@/components/Icons';
+
+export const toastCustom = (params: {
```

Comment:
Maybe call it `showToast` or something like this ? I understand that we don't want it to be called `toast` to not conflict with the `toast` from sonner; but showToast sounds better imo than `toastCustom` :)

Reply:
The problem is that we want something that starts with toast and that make you understand that it will use the toast.custom implementation. `showToast` does not match this unfortunately.